app-frontend/src/app/pages/projects/edit/aoi-approve/aoi-approve.html

```diff
@@ -0,0 +1,32 @@
+<div class="sidebar-project">
+  <div class="sidebar-header">
+    <a class="btn sidebar-header-nav-btn" ui-sref="projects.edit">
+      <i class="icon-arrow-left"></i>
+    </a>
+    <h5 class="sidebar-title">AOI Scene Approval</h5>
```

Comment:
In the original backend PR, it was requested that usage of the abbreviation `AOI` be changed to `area-of-interest` or similar. Thoughts on that here (and elsewhere?)